jetty-io/src/main/java/org/eclipse/jetty/io/ClientConnector.java

```
@@ -288,9 +301,34 @@ public void accept(SocketChannel channel, Map<String, Object> context)
         }
     }
 
+    private void bind(SocketChannel channel, SocketAddress bindAddress)
+    {
+        try
+        {
+            boolean reuseAddress = getReuseAddress();
+            if (LOG.isDebugEnabled())
+                LOG.debug("Binding to {} reusing address {}", bindAddress, reuseAddress);
+            channel.setOption(StandardSocketOptions.SO_REUSEADDR, reuseAddress);
+            channel.bind(bindAddress);
+        }
+        catch (Throwable x)
+        {
+            if (LOG.isDebugEnabled())
```

I'm not certain silently ignoring the bind address when bind fails is a sensible thing to do. Why do you want to change the existing behavior of throwing when bind fails?

---

Again because for Unix-Domain, binding makes no sense, so if `ClientConnector` is misconfigured, we ignore the binding attempt.